rero_ils/modules/utils.py

```diff
@@ -37,16 +37,16 @@ def strtotime(strtime):
     )
 
 
-def bulk_index(uuids, process=False, verbose=False):
+def do_bulk_index(uuids, doc_type=None, process=False, verbose=False):
```

Comment:
why did you change the name?

Reply:
Because it is not the bulk_index from the class but a function with bulk_index and reindex.